src/client/sandbox/code-instrumentation/index.ts

```diff
@@ -82,6 +86,26 @@ export default class CodeInstrumentation extends SandboxBase {
 
                 return html;
             },
+
+            configurable: true
+        });
+
+        nativeMethods.objectDefineProperty(window, INSTRUCTION.getProxyUrl, {
+            value: (url: any, baseUrl?: string) => {
+                const storedBaseUrl    = urlResolver.getBaseUrl(document);
+                const shouldChangeBase = baseUrl && baseUrl !== storedBaseUrl;
+
+                if (shouldChangeBase)
+                    urlResolver.updateBase(baseUrl, document);
+
+                const proxyUrl = getProxyUrl(String(url), { resourceType: stringifyResourceType({ isScript: true }) });
```

Comment:
`getProxyUrl(String(url),...`
We can move casting to string inside the `getProxyUrl` function.
```js
export function getProxyUrl (url: string, opts: ProxyUrlOptions) {
    url = String(url);
   
    const params: Array<string> = [opts.sessionId];
```

and then simplify all `getProxyUrl(String(url)` in our code.